test/scripts/e2e_subs/app-inner-calls-csp.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from goal import Goal
+import algosdk.logic as logic
+
+from datetime import datetime
+
+stamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+print(f"{os.path.basename(sys.argv[0])} start {stamp}")
+
+goal = Goal(sys.argv[1], autosend=True)
+
+joe = goal.new_account()
+
+txinfo1, err = goal.pay(goal.account, joe, amt=500_000)
```

Comment:
@algoidurovic Optional:  Since the return value is _not_ used, I propose explicitly signaling the intent by _not_ binding the unpacked tuple to a variable.

Aside:  The same concern exists in `app-inner-calls.py`.

```suggestion
_, err = goal.pay(goal.account, joe, amt=500_000)
```

Reply:
Seems there's a few additional places where a bound variable is unused.  Optionally, I propose unbinding all unused references (via `_`) in order to minimize confusion.

If you prefer to keep as is, then feel welcomed to resolve the thread.

Reply:
Sorry - The _only_ additional place is `txinfo1`.  

I saw several instances in `app-inner-calls.py` and mistakenly thought it was `app-inner-calls-csp.py`.